documentation/site/pages/provers/broker.mdx

```diff
@@ -162,6 +162,77 @@ cargo build --bin broker --release
 just broker down
```

Comment:
Should this step still list `just broker down` as the way to turn down your broker. It seems like this should warn provers and tell them to use the procedure just below when shutting down if actively proving on the network, to avoid losing stake.